src/stream.rs

```diff
@@ -8,7 +8,7 @@ use Compression;
 use ffi;
 
 pub struct Stream<D: Direction> {
-    raw: ffi::mz_stream,
+    pub raw: ffi::mz_stream,
```

Comment:
Ah, good point.  I fix that, renamed consumed_bytes to total_in for consistency, and added total_out for the hell of it.